hoot-js/src/main/cpp/hoot/js/schema/OsmSchemaJs.h

```diff
@@ -55,8 +55,13 @@ class OsmSchemaJs : public node::ObjectWrap
   static void getTagVertex(const v8::FunctionCallbackInfo<v8::Value>& args);
   static void isAncestor(const v8::FunctionCallbackInfo<v8::Value>& args);
 
-  // Even though all of the logic for these methods have been moved from OsmSchema to criterion,
+  static void score(const v8::FunctionCallbackInfo<v8::Value>& args);
+  static void scoreOneWay(const v8::FunctionCallbackInfo<v8::Value>& args);
+
+  // Even thoughthe logic for some of these methods have been moved from OsmSchema to criterion,
   // decided to leave this interface intact, as it will be simpler to access from js.
+
+  // TODO: All of these methods can go away if #3047 is completed.
```

Comment:
I grouped all of these methods together that we won't really need anymore if #3047 is completed.  We'll still support isMatchCandidate checks in Generic Conflation scripts, but specifying the criterion will be good enough in most cases.